### PR TITLE
Feature/activity feed

### PR DESCRIPTION
**Pull request by cvializ**
@andrewconner (@adamjgrant) https://app.asana.com/0/42816489233171/44164549287817

### CODE REVIEW

**Comment by cvializ** [comment]
screen shot 2015-08-25 at 2 42 21 pm

**Comment by cvializ** [comment]
@andrewconner :8ball: I see a PR review in our future

File: kifi-backend/modules/shoebox/angular/src/route.js [hunk 1]
```diff
@@ -26,6 +26,26 @@ angular.module('kifi')
     $stateProvider
       .state('home', {  // Home page.
         url: '/',
+        controller: ['$state', 'me', function ($state, me) {
+          var showFeed = (me.orgs.length > 0);
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/route.js]: Could we deploy to ourselves first (check experiments)? Would let us make sure everything's good before switching some new users over.

File: kifi-backend/modules/shoebox/angular/dev.html [hunk 2]
```diff
@@ -14,6 +14,15 @@
 <div class="kf-app ng-cloak" ng-controller="AppCtrl">
   <div ng-if="$root.userLoggedIn != null" ng-include="'layout/body.tpl.html'"></div>
 </div>
+
+<script>
+  // Uncomment this to see micrologger reports in the console.
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/dev.html:18]: Should this be commented then?

File: kifi-backend/modules/shoebox/angular/src/feed/feedService.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+angular.module('kifi')
+
+.factory('feedService', [
+  '$http', '$rootScope', 'profileService', 'routeService', '$q', '$analytics', 'net', 'ml',
+  function ($http, $rootScope, profileService, routeService, $q, $analytics, net, ml) {
+
+    var api = {
+      getFeed: function (limit, beforeId, afterId) {
+        ml.specs.getsFeed = new ml.Spec([
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/feed/feedService.js:11]: Interesting. I saw @adamjgrant's commit about this.

File: kifi-backend/modules/shoebox/angular/dev.html (hunk 2 above)
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/dev.html:18]: This is in `dev.html`, I think @adamjgrant intended for the comment to be in `index.html`

**Comment by andrewconner** [comment]
:+1: Looks good! I know soon it'll be for all orgs-users, but would you like to do a smaller set (just us) first?

File: kifi-backend/modules/shoebox/angular/src/route.js (hunk 1 above)
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/route.js]: Naturally. Should I create a new experiment, or should I just make it visible to 'fake' and 'admin'?

File: kifi-backend/modules/shoebox/angular/dev.html (hunk 2 above)
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/dev.html:18]: Oh okay. Do we do anything in index.html?

File: kifi-backend/modules/shoebox/angular/src/route.js (hunk 1 above)
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/route.js]: The latter sounds good for me, unless you want it smaller than just us?

Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/route.js]: Awesome, just us is fine, but I wanted to double check

File: kifi-backend/modules/shoebox/angular/dev.html (hunk 2 above)
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/dev.html:18]: Not yet, I'll add it in, but commented

**Pull request merged**